fixtures/etc/fos4PL8.json

```diff
@@ -0,0 +1,368 @@
+    ],
+    "productPage": [
+      "https://www.etcconnect.com/Products/Lighting-Fixtures/fos/4-Panels/Features.aspx"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=VTWd9qrpuW8"
+    ]
+  },
+  "physical": {
+    "dimensions": [203, 610,  631],
+    "weight": 14.5,
+    "power": 540,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Intensity": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Deep Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "Lime": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Lime"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Cyan": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Cyan"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Indigo": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Indigo"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [1, 40],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "1Hz",
+          "speedEnd": "40Hz",
+          "comment": "Dark Strobe"
+        },
+        {
+          "dmxRange": [41, 80],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "1Hz",
+          "speedEnd": "40Hz",
+          "comment": "Bright Strobe"
+        },
+        {
+          "dmxRange": [81, 120],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse"
+        },
+        {
+          "dmxRange": [121, 160],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse",
+          "comment": "Random"
+        },
+        {
+          "dmxRange": [161, 200],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Spikes",
+          "comment": "Flicker"
+        },
+        {
+          "dmxRange": [201, 240],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [241, 254],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        }
+      ]
+    },
+    "Curve": {
```

Comment:
Please rename this channel to "Dimmer Curve" to match similar channels in other fixtures.